src/data/markdown/docs/40 xk6-disruptor/03 Examples.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'Examples'
+excerpt: 'Examples of how to use the xk6-disruptor extension to introduce faults in k6 tests.'
+---
+
+In the next sections we present some examples of how to use the `xk6-disruptor` extension to introduce faults in `k6` tests. Some examples make use the [k6-kubernetes](http://github.com/grafana/xk6-kubernetes) extension. Ensure the custom `k6` binary you are using is built with both `xk6-disruptor` and `xk6-kubernetes` extensions. See the [Installation guide](../01-get-started/03-installation.md) for details.
```

Comment:
```suggestion
In these sections we present some examples of how to use the `xk6-disruptor` extension to introduce faults in `k6` tests.
Some examples use the [k6-kubernetes](http://github.com/grafana/xk6-kubernetes) extension.
So, to follow, ensure the custom `k6` binary you are using is built with both `xk6-disruptor` and `xk6-kubernetes` extensions. See the [Installation guide](../01-get-started/03-installation.md) for details.
```

Reply:
Not sure about "these" instead of "the next". I thought "these" always refereed to something previously mentioned.